### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: doc/ADMIN.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Configuration
+
+Edit the file `__INSTALL_DIR__/lutim.conf` to configure Lutim.
+Then reload the lutim service.
```
Comment (alexAubin) [inline doc/ADMIN.md:4]:
```suggestion
You can edit `__INSTALL_DIR__/lutim.conf` to configure Lutim, then reload the `lutim` service.
```

File: doc/ADMIN_fr.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Éditez le fichier `__INSTALL_DIR__/lutim.conf` pour configurer Lutim.
+Rechargez ensuite le service Lutim.
```
Comment (alexAubin) [inline doc/ADMIN_fr.md:2]:
```suggestion
Vous pouvez modifier `__INSTALL_DIR__/lutim.conf` pour configurer Lutim, puis rechargez le service `lutim`.
```

File: scripts/upgrade [hunk 3]
```diff
@@ -76,43 +36,13 @@ ynh_systemd_action --service_name=$app --action="stop"
 #=================================================
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=2
 
-skipped_uris=$(ynh_app_setting_get --app=$app --key=skipped_uris)
+#REMOVEME? skipped_uris=$(ynh_app_setting_get --app=$app --key=skipped_uris)
 
 # Unused with the permission system
 if [ ! -z "$skipped_uris" ]; then
 	ynh_app_setting_delete --app=$app --key=skipped_uris
 fi
```
Comment (alexAubin) [inline scripts/upgrade]:
```suggestion
```

**Pull request merged**